docs/roadmap.md

```diff
@@ -3,15 +3,82 @@
 ## Table of Contents
 - [Status Roadmap](#status-roadmap)
   - [Table of Contents](#table-of-contents)
-  - [2025 H2](#2025-h2)
-    - [2.35](#235)
+  - [2026 H1](#2026-h1)
+    - [2.38](#238)
       - [Features](#features)
-    - [2.36](#236)
+    - [2.39](#239)
       - [Features](#features-1)
-    - [2.37](#237)
+  - [2025 H2](#2025-h2)
+    - [2.35](#235)
       - [Features](#features-2)
-    - [2.38](#238)
+    - [2.36](#236)
       - [Features](#features-3)
+    - [2.37](#237)
+      - [Features](#features-4)
+
+## 2026 H1
+
+### 2.38
+
+Release Epic: https://github.com/status-im/status-desktop/issues/19509
+
+Estimated release: Early February
+
+#### Features
```

Comment:
We already have the **Activity Center** [epic](https://github.com/status-im/status-app/issues/18516) planned for the `2.37` release (but not part of this list), with some work that actually started earlier and was partially delivered in `2.36`. I would recommend prioritising this and bringing it to completion, as we currently have a “half-feature” in place and the overall UX is not yet where it should be.

Reply:
Ah, yes. I cleaned 2.37 already and moved it to 2.38, but forgot to add it to this list (it wasn't part of the ones we got to during our prioritization meeting.

I'll add it.

Reply:
Yeah, this doesn't need more design, since it's almost done, so I didn't add it to the meeting discussion.. true :face_with_peeking_eye: